scripts/travis/run_test.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
+    scripts/travis/osx/build.sh
+    scripts/travis/print_status.sh rocproject/ci-osx $?
```

Comment:
Do we really need print_status for os x builds? I think it makes sense only for linux builds, where we test many docker containers in a single travis build.

Reply:
You are right, I thought that we would have various osx versions, but each of them will be a single travis build.